go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	github.com/grafana/regexp v0.0.0-20220304100321-149c8afcd6cb
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
-	github.com/hashicorp/consul/api v1.13.0
+	github.com/hashicorp/consul/api v1.15.2
```

Comment:
Are all these version changes needed for oltp? If not I'd prefer it if they were in a separate PR.